backend/routes/auth.js

```diff
@@ -543,22 +547,45 @@ router.put("/:account", middlewares.isAccountAdmin, updateUser);
 router.put("/:account/password", resetPassword);
 
 function createSession(place, req, res, next, user) {
-
 	req.session.regenerate(function(err) {
 		req[C.REQ_REPO].logger.logInfo("Creating session for " + " " + user.username);
 		if(err) {
 			responseCodes.respond(place, responseCodes.EXTERNAL_ERROR(err), res, {username: user.username});
 		} else {
 			req[C.REQ_REPO].logger.logDebug("Authenticated user and signed token.");
 
-			req.session[C.REPO_SESSION_USER] = user;
-			req.session.cookie.domain				 = config.cookie_domain;
+			user.socketId = req.headers[C.HEADER_SOCKET_ID];
+			user.webSession = false;
```

Comment:
```suggestion
			user.webSession = req.headers && req.headers["user-agent"]
```
I think that should work and we can ditch 560-562?

That sounds like a massive assumption though to say anyone who sends http request with user-agent filled in = browser.... @sanmont3drepo  any thoughts?